lib/routes/latepost/index.ts

```diff
@@ -72,7 +90,7 @@ async function handler(ctx) {
     let items = response.data.slice(0, limit).map((item) => ({
         title: item.title,
         link: new URL(item.detail_url, rootUrl).href,
-        category: [item.is_dj ? exclusiveCategory : undefined, item.programa ? columns[item.programa].title : undefined, ...item.label.map((c) => c.label)],
+        category: [item.is_dj ? exclusiveCategory : undefined, itemCategory(item.programa), ...item.label.map((c) => c.label)],
```

Comment:
The object key from 

https://github.com/DIYgod/RSSHub/blob/687f76247ffb95432523c5b523ebbf9c674887ff/lib/routes/latepost/index.ts#L80

is already a string.

```ts
{
  '1': { title: '晚点独家', web_url: '/news/index?proma=1' },
  '2': { title: '人物访谈', web_url: '/news/index?proma=2' },
  '3': { title: '晚点早知道', web_url: '/news/index?proma=3' },
  '4': { title: '长报道', web_url: '/news/index?proma=4' }
}
```

The error occurs simply because the code is trying to call `columns['5'].title`


```suggestion
        category: [item.is_dj ? exclusiveCategory : undefined, item.programa ? columns[item.programa]?.title : undefined, ...item.label.map((c) => c.label)],
```

Reply:
明白了， 我重新发一个 pr